.ci/check-tidy

```diff
@@ -57,5 +57,5 @@ for dir in build build-build; do
 	echo "Checking $(echo ${SOURCES} | wc -w) files with clang-tidy"
 
 	# The list of tidy checks must stay in sync with scripts/tidy
-	${CLANGTIDY} -quiet -p ${dir} -checks=-*,clang-analyzer-*,-clang-analyzer-cplusplus*,bugprone-*,performance-*,portability-*,readability-*,-readability-braces-around-statements,-readability-magic-numbers,-readability-isolate-declaration,-readability-identifier-length,-readability-function-cognitive-complexity,-bugprone-reserved-identifier,-bugprone-easily-swappable-parameters,-performance-no-int-to-ptr -warnings-as-errors=* -header-filter '.*' ${SOURCES}
+	${CLANGTIDY} -quiet -p ${dir} -checks=-*,clang-analyzer-*,-clang-analyzer-cplusplus*,bugprone-*,performance-*,portability-*,readability-*,-readability-braces-around-statements,-readability-magic-numbers,-readability-isolate-declaration,-readability-identifier-length,-readability-function-cognitive-complexity,-bugprone-reserved-identifier,-bugprone-easily-swappable-parameters,-bugprone-narrowing-conversions,-performance-no-int-to-ptr -warnings-as-errors=* -header-filter '.*' ${SOURCES}
```

Comment:
Needed to change this because I was getting errors like this all over the `ui_images.c` file.
```
/bitbox02-firmware/src/ui/components/ui_images.c:26:25: error: narrowing conversion from 'int' to signed type 'UG_S16' (aka 'short') is implementation-defined [bugprone-narrowing-conversions,-warnings-as-errors]
            UG_DrawLine(x + h, y + h, x + h, y + width - 1 - h, C_WHITE);
```

Reply:
That is sth for the commit msg :)